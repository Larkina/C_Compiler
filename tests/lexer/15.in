switch(k) {
	case 1:
	{
	 /* unclosed comment
	}
}