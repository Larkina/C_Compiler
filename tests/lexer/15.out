value=switch, text=switch, type=VAR
value=(, text=(, type=L_PARENTHESIS
value=k, text=k, type=VAR
value=), text=), type=R_PARENTHESIS
value={, text={, type=L_BRACE
value=case, text=case, type=VAR
value=1, text=1, type=INT
value=:, text=:, type=COLON
value={, text={, type=L_BRACE
Incorrect token on 6 line, 6 pos: Unclosed multiline comment